### DISCUSSION TITLE
Add date (time) criteria to sync only files newer (or older) than a specific time

### DISCUSSION THREAD

**Discussion by zOuateRabbit** [RsyncProject/rsync#617] [Ideas]
Add date (time) criteria to sync only files newer (or older) than a specific time

A bit like the find command, it would be good to have a date criteria like that:

find . -newermt now-25hours

**Comment by ghost** [comment]
> Add date (time) criteria to sync only files newer (or older) than a specific time
> 
> A bit like the find command, it would be good to have a date criteria like that:
> 
> find . -newermt now-25hours

STFU
